### DISCUSSION TITLE
THS01 Supported?

### DISCUSSION THREAD

**Discussion by jonwedell** [arendst/Tasmota#16279] [Q&A] [answered]
I have a [THR316D](https://itead.cc/product/sonoff-th-elite-smart-temperature-and-humidity-monitoring-switch/) with a [THS01](https://itead.cc/product/sonoff-ths01-temp-and-humi-sensor/) temperature and humidity sensor. I have the latest release of Tasmota installed, and I'm unable to get the THS01 working.

It was my understanding the THS01 is just a rebranded AM2301 - which I expected to work with Tasmota. Does anyone know if this combination is supported? If so, any ideas what I might have been doing wrong? I tried setting AM2301 on GPIO25 to no avail, but I also tried every free GPIO just to ensure I didn't have the pin wrong - all without success. Thanks!

**Comment by sfromis** [comment, marked as answer]
Reading https://github.com/arendst/Tasmota/issues/15856 suggests that you may have to config it as "SI7021"

**Comment by jonwedell** [comment]
Awesome, that did it! I read through that whole thread but am new enough to Tasmota to have not understood that was the appropriate solution. Thanks!
